spec/models/form/lettings/questions/stock_owner_spec.rb

```diff
@@ -205,21 +208,26 @@
     end
 
     context "when user is support" do
-      let(:user) { create(:user, :support) }
+      user = nil
+      log = nil
+
+      before do
+        user = create(:user, :support)
+        log = create(:lettings_log)
+      end
 
-      let(:log) { create(:lettings_log) }
+      it "shows active orgs where organisation holds own stock" do
+        non_stock_organisation = create(:organisation, name: "Non-stockholding org", holds_own_stock: false)
+        inactive_organisation = create(:organisation, name: "Inactive org", active: false)
 
-      let(:non_stock_organisation) { create(:organisation, holds_own_stock: false) }
```

Comment:
The timing here meant that this organisation didn't actually exist at the relevant point in the tests.